lib/cisco_node_utils/cmd_ref/radius_server.yaml

```diff
@@ -1,66 +1,101 @@
 # radius_server.yaml
 ---
-_exclude: [ios_xr]
+_template:
+  ios_xr:
+    get_command: "show running-config radius-server"
+    multiple: true
 
 accounting:
+  _exclude: [ios_xr]
```

Comment:
Would it make sense at all for this to be `default_only: true` for XR instead of excluded? Not sure myself.